system/churn/playlist.xml

```diff
@@ -79,14 +47,11 @@
 			</disable>
 		</disables>
 		<command> 
-            export OTOOL_JDK_VERSION="$(JDK_VERSION)" ; \
             export JREJDK="jdk" ; \
             export OTOOL_garbageCollector="${CHURN_GCS}" ; \
-            if [ "0${CHURN_DURATION}" -gt 86400 ] ; then CHURN_DURATION=86400  ; fi ; \
             export DURATION="${CHURN_DURATION}" ; \
-            export TMPRESULTS="$(Q)$(REPORTDIR)$(D)report$(Q)"; \
-            cd $TMPRESULTS ; \
-              bash "$(TEST_ROOT)$(D)system$(D)churn$(D)churn$(D)run.sh"; \
+            if [ "x${CHURN_TAP}" = "x" ] ; then export CHURN_TAP=true ; fi ; \
```

Comment:
What's the `CHURN_TAP` ?

Reply:
A tap file that Jiri generates himself, which does not conform to the same information as the tap file that TKG generates, so I asked that it not be generated when running as part of AQAvit, as we use the TKG formated tap files for many purposes and having a non-conformant tap file on the system would be sub-optimal.

See comments in original PR [starting here](https://github.com/adoptium/aqa-tests/pull/5101#issuecomment-1971071239) for background.

Reply:
CHURN_TAP is control variable, which inside churn runner generate tap file - if empty or true. If it is set an non true, then tap file is not generated.

The tap file from churn is extremely useful so I let it on for the custom manual run.  However  it disturbed the automated tuns,  so it is off for default auto run.

If you insists, I will turn it off also for manual run. 

The condition is here, so the user from cmdline can control the behaviour.

Reply:
Worth a comment in the code if this causes confusion